examples/requirements.txt

```diff
@@ -12,3 +12,7 @@ hdbscan
 matplotlib
 seaborn
 tika
+gensim
+-e https://github.com/zelandiya/RAKE-tutorial.git
```

Comment:
This line should be: `-e git://github.com/zelandiya/RAKE-tutorial.git#egg=nlp-rake`
Otherwise it is throwing me errors.

Reply:
fixed